### PR TITLE
#75 implement comment buttons

### LINKED ISSUE #75: Implement buttons functionality on comments.
save state and visualize status to user. Position is not to be implanted yet!

### PR DESCRIPTION
**Pull request by FredSoderberg**
#### What does this PR do?
Adds functionality to comment buttons. To some extent also the api calls which are to be fully implemented in #72. There is also some null checks for items and comment components.
[#75](LINK_TO_ISSUE)

##### Why are we doing this? Any context or related work?
Steps to implement the gamification part

#### Where should a reviewer start?
comment component

#### Put `closes #XXXX` in your comment to auto-close the issue that your PR fixes (if such).
closes #75

### CODE REVIEW

File: Bikeini/components/Comment.js [hunk 1]
```diff
@@ -78,16 +79,66 @@ const styles = StyleSheet.create({
...
+    };
+  }
+
+  sendCommentPoints = (points) => {
+    const { author, jwt } = this.props;
+    const formBody = {
+      author, points,
+    };
+
+    console.log(jwt, formBody);
```
Comment (erikajansson1) [inline Bikeini/components/Comment.js]: Console.log

File: Bikeini/components/Comment.js [hunk 2]
```diff
@@ -104,10 +155,42 @@ export default class Comment extends React.PureComponent {
...
+            style={styles.locationTag}
+            source={locationIcon}
+          />
+        </TouchableOpacity>
+        <TouchableOpacity
+          style={styles.thumbDownTag}
+          onPress={() => this.handleThumbs('DW')}
+        >
+          <Image
+            style={[styles.thumbDownTag, thumbDown ? styles.setRed : []]}
```
Comment (erikajansson1) [inline Bikeini/components/Comment.js:170]: Nice with change of color!

**Comment by erikajansson1** [comment]
Well done! I think it looks good! Nice "cleanup" of _ -> Camelcase and so on!

File: Bikeini/components/Comment.js (hunk 1 above)
Comment (FredSoderberg) [reply Bikeini/components/Comment.js]: ok

Comment (FredSoderberg) [reply Bikeini/components/Comment.js]: ok

**Pull request merged**